src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -19,12 +19,14 @@
 
 import static com.jayway.awaitility.Awaitility.await;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
 import static org.hamcrest.Matchers.is;
 
 public class ProcessedEnvelopeMessageHandlingTest extends BaseFunctionalTest {
 
     private static final long MAX_MESSAGE_PROCESSING_TIME_MILLIS = 40_000;
     private static final long MAX_ENVELOPE_FINALISATION_TIME_MILLIS = 10_000;
+    private static final int DELETE_TIMEOUT = 20_000;
```

Comment:
Could we name it using the same convention as the two other constants, i.e. `MAX_BLOB_DELETION_TIME_MILLIS` or sth like that?

Reply:
@lgonczar done